examples/self_weight_clamped_truss/self_weight_clamped_truss.jl

```diff
@@ -51,7 +51,7 @@ function analytic_u(x::Real; F::Real, E::Real,
 end
 
 "Return mesh"
-function mesh(N::Int, L::Real, A::Real, ϵ_model)
+function create_mesh(N::Int, L::Real, A::Real, ϵ_model)
```

Comment:
Makes sense, so yo don't have to prepend with the module's name each time.

Reply:
Yep, thanks!